packages/utils/src/carequality/cq-monthly-stats/shared.ts

```diff
@@ -0,0 +1,140 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { initReadonlyDbPool } from "@metriport/core/util/sequelize";
+import { QueryTypes } from "sequelize";
+
+const sqlDBCreds = getEnvVarOrFail("DB_CREDS");
+const sqlReadReplicaEndpoint = getEnvVarOrFail("DB_READ_REPLICA_ENDPOINT");
+
+export const readOnlyDBPool = initReadonlyDbPool(sqlDBCreds, sqlReadReplicaEndpoint);
+
+export type RequestParams = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  cqDirectory: any[];
```

Comment:
Since this is a script that will be ran once a month i didnt find it necessary to export the cqDirectory types to a core package from api so that it could be used here